sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -279,21 +292,23 @@ def _fit(self, y, X, fh):
                 "peft",
                 severity="error",
                 msg=(
-                    f"Error in {self.__class__.__name__}: 'peft' module not found. "
-                    "'peft' is a soft dependency and not included "
-                    "in the base sktime installation. "
-                    "To use this functionality, please install 'peft' by running: "
-                    "`pip install peft` or `pip install sktime[dl]`. "
-                    "To install all soft dependencies, "
-                    "run: `pip install sktime[all_extras]`"
+                    "PEFT package is required but not installed. Install using:\n"
+                    "`pip install peft`\n\n"
+                    "Additionally, if you want to all optional dependencies for sktime,"
+                    "run:\n"
+                    "`pip install sktime[all_extras]`.\n\n"
+                    "For more details on PEFT, visit the official documentation at:\n"
+                    "https://huggingface.co/docs/peft."
                 ),
             ):
                 from peft import get_peft_model
-            peft_config = deepcopy(self.peft_config)
-            self.model = get_peft_model(self.model, peft_config)
+
+                peft_config = deepcopy(self.peft_config)
+                self.model = get_peft_model(self.model, peft_config)
```

Comment:
I would suggest to keep these lines outside the dependency `if block`, it doesnot really change anything, but just a common practise
```suggestion

          peft_config = deepcopy(self.peft_config)
          self.model = get_peft_model(self.model, peft_config)
```